pkg/ebpf/co_re.go

```diff
@@ -42,6 +42,22 @@ func LoadCOREAsset(filename string, startFn func(bytecode.AssetReader, manager.O
 	return loader.loadCOREAsset(filename, startFn)
 }
 
+// Returns where the ebpf BTF files were sourced from
```

Comment:
typically these comments include the method name as the first word

Reply:
Ah good callout